src/geo/ui/legends/base/legend-title.tpl

```diff
@@ -1 +1 @@
-<h3 class="CDB-Text CDB-Size-small u-upperCase u-bSpace--m u-altTextColor"><%= title %></h3>
+<h3 class="CDB-Text CDB-Size-small u-upperCase u-bSpace u-altTextColor"><%= title %></h3>
```

Comment:
Now that I see it... we should use `<%- title %>` here. Would you please change it for me?